docs/questions/native-editor/basic-sql-parameters.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Basic SQL parameters
+summary: Text, number, and date variables let you plug basic values into your SQL code.
```

Comment:
Can we also add the summary as the first sentence in the article? (or the first sentence after the blockquote)